### ISSUE TITLE
React Storybook (et angular Playground)

### ISSUE THREAD

**Issue by 4ian** [dahfazz/RD-dej-front#2]
React Storybook en quelques points :

- ce que c'est :smile:
 - l'avantage principal : accélerer le développement des composants visuels,
 - le second avantage : vous aider à faire valider à votre PO des composants en dehors de votre app et donc découper plus facilement vos user stories,
 - le troisieme avantage : une documentation automatique des composants réutilisables de votre app,
 - comment l'intégrer en 10 minutes sur votre projet

**Comment by evanliomain** [comment]
J'ai vu la présentation de React Storybook à BestOfWeb 2017. C'est terrible cet outil ! 

C'est un besoin que j'ai déjà essayé de recréé manuellement sur 2 projets (angularJS et angular).
Je ne connaissais pas la version angular. Le nommage de ces outils est tellement disparate (styleguide, storybook, playground).
Angular material utilise dgeni en interne. Mais c'est très lourd et mal documenté pour le mettre en place.

**Comment by 4ian** [comment]
Yes, je l'utilise sur un projet et rien qu'avec ce qu'on a commencé à faire dessus, on a gagné pas mal de temps :) Chaud du coup pour présenter ça un midi à Renault Digital
